injector/dns_disruption.go

```diff
@@ -229,8 +215,29 @@ func (i *DNSDisruptionInjector) Clean() error {
 		}
 	}
 
-	if err := i.config.Iptables.ClearAndDeleteChain("CHAOS-DNS"); err != nil {
-		return fmt.Errorf("unable to remove injected iptables chain: %w", err)
+	// check if there's still a jump rule to CHAOS-DNS chain existing
+	// that would block its deletion
```

Comment:
What is the leftover jump rule? Doesnt this risk leaving this chain around?

Reply:
There's one rule per injector and injectors are cleaning their own rule sequentially so there's no risk of a remaining rule skipping the chain deletion unless one injector fails to clean its own rule which would lead to a stuck disruption anyway.